testbed/tests/log_test.go

```diff
@@ -105,7 +105,7 @@ func TestLog10kDPS(t *testing.T) {
 			sender:   datasenders.NewSyslogWriter("udp", testbed.DefaultHost, testbed.GetAvailablePort(t), 1),
 			receiver: testbed.NewOTLPDataReceiver(testbed.GetAvailablePort(t)),
 			resourceSpec: testbed.ResourceSpec{
-				ExpectedMaxCPU: 80,
+				ExpectedMaxCPU: 200,
```

Comment:
Are we able to identify what part of this change has caused the additional resource utilization?

Reply:
I found this: https://github.com/open-telemetry/opentelemetry-log-collection/pull/161